### PR TITLE
AC: Added command line argument model_attributes for vocab and color_coeffs path's prefix

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

**Comment by Anna-Mironova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/README.md [hunk 1]
```diff
@@ -88,21 +88,23 @@ All relative paths in config files will be prefixed with values specified in com
...
 - `--num_requests` number requests for async execution. Allows override provided in config info. Default is `AUTO`
+- `--model_attributes` prefix for path with additional models attributes.
 
 You are also able to replace some command line arguments with environment variables for path prefixing. Supported following list of variables:
 * `DATA_DIR` -  equivalent of `-s`, `--source`.
 * `MODELS_DIR` - equivalent of `-m`, `--models`.
 * `EXTENSIONS` - equivalent of `-e`, `--extensions`.
 * `ANNOTATIONS_DIR` - equivalent of `-a`, `--annotations`.
 * `BITSTREAMS_DIR` - equivalent of `-b`, `--bitstreams`.
+* `ATTRIBUTES_DIR` - equivalent of `--model_attributes`.
```
Comment (eaidova) [inline tools/accuracy_checker/README.md:107]: I think better to name it as MODEL_ATTRIBUTES_DIR

File: tools/accuracy_checker/README.md [hunk 2]
```diff
@@ -88,21 +88,23 @@ All relative paths in config files will be prefixed with values specified in com
...
-- `-d, --definitions` path to the global configuration file
+- `-d, --definitions` path to the global configuration file.
 - `-e, --extensions` directory with InferenceEngine extensions.
 - `-b, --bitstreams` directory with bitstream (for Inference Engine with fpga plugin).
 - `-C, '--converted_models` directory to store Model Optimizer converted models (used for DLSDK launcher only).
 - `-tf, --target_framework` framework for infer.
 - `-td, --target_devices` devices for infer. You can specify several devices using space as a delimiter.
 - `--async_mode` allows run the tool in async mode if launcher support it.
 - `--num_requests` number requests for async execution. Allows override provided in config info. Default is `AUTO`
+- `--model_attributes` prefix for path with additional models attributes.
```
Comment (eaidova) [inline tools/accuracy_checker/README.md:99]:
```suggestion
- `--model_attributes` directory with additional models attributes
```

File: tools/accuracy_checker/README.md [hunk 3]
```diff
@@ -88,21 +88,23 @@ All relative paths in config files will be prefixed with values specified in com
 You may refer to `-h, --help` to full list of command line options. Some optional arguments are:
 
 - `-r, --root` prefix for all relative paths.
```
Comment (eaidova) [inline tools/accuracy_checker/README.md:90]: hmm... we do not have this command line parameter at all :)

File: tools/accuracy_checker/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/README.md]: Done

File: tools/accuracy_checker/README.md (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/README.md]: Done

File: tools/accuracy_checker/README.md (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/README.md]: Parameter -r has been removed from description

**Pull request merged**